### PR TITLE
doc: Add antenna types

### PR DESCRIPTION
**Pull request by pradeeka7**
#### Summary

This study material includes some of the popular antenna types used with LoRaWAN end devices and gateways.

#### Screenshots

image

#### Changes

- ...
- ...

#### Notes for Reviewers

@benolayinka Can you review?

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [X] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/lorawan/antenna-types/_index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+In this tutorial, you will learn about some of the popular antenna types used with LoRaWAN end devices and gateways.
+
+## Wire Antenna
+
+Wire antennas are low-cost and easy to build and work well in indoor environments. They are usually quarter-wave antennas. Different frequencies need different wire lengths (calculated for quarter-wave), for example, 3.25 inches or 8.2 cm for 868 MHz and 3 inches or 7.8 cm for 915 MHz. Suitable for DIY projects. You must keep the antenna wire as vertical as possible for better performance.
+
+{{< figure src="wire-antenna-example.jpg" class="plain" >}}
+image credits: [SparkFun Electronics](https://learn.sparkfun.com/)
```
Comment (nejraselimovic) [inline doc/content/lorawan/antenna-types/_index.md]: If you use _image courtesy of ..._ in one page, please be consistent and use the same in others as well

File: doc/content/lorawan/antenna-types/_index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+## PCB Antenna
+
+These antennas are made from a copper trace drawn directly onto a PCB. Great for high volume production and suitable for indoor use. 
+
+{{< figure src="pcb-antenna-example.png" class="plain" >}}
+image credits: [DesignSpark](https://www.rs-online.com/designspark/things-network-hardware-makes-lorawan-easy)
+
+## Spring Antenna
+
+Spring antennas (also known as Coil/Helical antennas) are made from coiled wires (usually copper or aluminium) that reduce the antenna’s length. These antennas are suitable for use with LoRa modules with low transmission power (up to 100mW) and are also perfect for end devices with space constraints.
```
Comment (nejraselimovic) [inline doc/content/lorawan/antenna-types/_index.md]:
```suggestion
Spring antennas (also known as coil/helical antennas) are made from coiled wires (usually copper or aluminium) that reduce the antenna’s length. These antennas are suitable for use with LoRa modules with low transmission power (up to 100mW) and are also perfect for end devices with space constraints.
```

File: doc/content/lorawan/antenna-types/_index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Rubber Duck Antenna
+
+These antennas are made from rubber/plastic (housing) and copper/aluminium (actual antenna). Rubber duck antennas are usually half wave or quarter wave. They terminated with SMA or RP-SMA connector for directly connecting with the gateway/end device or for connecting through a pigtail. Ideal for LoRa nodes and indoor gateways.
+
+{{< figure src="rubber-duck-antenna-example.jpg" class="plain" >}}
+image credits: [SparkFun Electronics](https://learn.sparkfun.com/)
+
+## Collinear Antenna
+
```
Comment (nejraselimovic) [inline doc/content/lorawan/antenna-types/_index.md:38]: collinear antenna, not Colinear antenna

**Review by nejraselimovic — CHANGES_REQUESTED** [review]

**Comment by pradeeka7** [comment]
@nejraselimovic done

**Pull request merged**